enterprise/backend/src/metabase_enterprise/workspaces/api.clj

```diff
@@ -245,6 +306,37 @@
   (let [_ [id body]] ;; TODO stub
     1))
 
+(api.macros/defendpoint :post "/:id/transform"
```

Comment:
I wonder whether we don't want a generic route for any kind of entity. This feels more coherent with the other APIs.

Will make it easy to write data-oriented code later, versus lots of boilerplate.

Perhaps from an OpenAPI perspective we prefer simpler special purpose routes, I'm not fully decided.